starboard/android/shared/log_is_tty.cc

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if SB_API_VERSION < 16
+#include "starboard/log.h"
+
+bool SbLogIsTty() {
```

Comment:
What code is using `SbLogIsTty`?

Reply:
It's not actually used in a meaningful way, but it's still expected to be an exported symbol for SB 14/15:
https://github.com/youtube/cobalt/blob/25.lts.1%2B/starboard/elf_loader/exported_symbols.cc#L205. So Evergreen-AOSP crashes when loading Cobalt Core on SB 14/15.

I think in general we should always use SB version guards when removing Starboard API implementations.